### DISCUSSION TITLE
High MOI control group complement set sgRNA specifics

### DISCUSSION THREAD

**Discussion by yabeykoon** [Katsevich-Lab/sceptre#137] [Q&A] [answered]
When SCEPTRE assigns a complement set as the control group in a high MOI perturb-seq analysis where multiple sgRNA target a specific candidate enhancer element, especially in singleton and Bonferroni sgRNA assignment methods when a single sgRNA targeting a specific element is analyzed in treatment cells compared to the control, does SCEPTRE exclude cells containing other sgRNA targeting that same element from the complement set? I assume since union is element level, this would not be an issue.

Additionally if I have two separate candidate enhancer elements regulating the same gene, is there a way to exclude sgRNA targeting that second element from the complement set that might dampen the treatment signal from the sgRNA targeting the first element?

Thank you!

**Comment by timothy-barry** [comment, marked as answer]
Hi apologies for the delayed response, it's been a bit busy recently. :)

> When SCEPTRE assigns a complement set as the control group in a high MOI perturb-seq analysis where multiple sgRNA target a specific candidate enhancer element, especially in singleton and Bonferroni sgRNA assignment methods when a single sgRNA targeting a specific element is analyzed in treatment cells compared to the control, does SCEPTRE exclude cells containing other sgRNA targeting that same element from the complement set? I assume since union is element level, this would not be an issue.

No, cells containing other gRNAs targeting the same site are not excluded from the complement set. Our thought process is that the total number of cells in the complement set is much greater than the number of cells harboring a gRNA targeting the same site. Thus, cells containing a gRNA targeting the same site should be "swamped" by the other complement set cells.

> Additionally if I have two separate candidate enhancer elements regulating the same gene, is there a way to exclude sgRNA targeting that second element from the complement set that might dampen the treatment signal from the sgRNA targeting the first element?

Unfortunately, this is not currently possible. Again, our hope is that the "swamping" phenomenon described above should largely neutralize this issue.

The simple definition for "complement set" that we are currently using enables us to recycle computation across hypotheses, thereby substantially reducing compute time. I think the statistical price that we pay for this decision is negligible.

**Comment by yabeykoon** [comment]
Thank you! That makes sense. When we compare the treatment vs control group for a specific sgRNA, is the number of cells in each group determined by the expression level of the gene that it's paired with? I'm curious to know what makes the cell numbers different for the same exact sgRNA when paired with different responses.

**Comment by timothy-barry** [comment]
> When we compare the treatment vs control group for a specific sgRNA, is the number of cells in each group determined by the expression level of the gene that it's paired with?

Not exactly. In high-MOI, the number of cells used per test is actually the exact same *across all tests*. Are you referring to the `n_nonzero_trt` and `n_nonzero_cntrl` columns? Those do not refer to the number of cells, but rather the number of cells with nonzero expression of the gene. See [here](https://timothy-barry.github.io/sceptre-book/run-qc.html#sec-qc_pairwise_qc) for more info and please feel free to ask additional questions if this is unclear.
